### PR TITLE
React virtualized

### PR DESCRIPTION
**Pull request by shalabhsuman**

### CODE REVIEW

File: src/pages/recentDeliveries/RecentDeliveriesPage.tsx [hunk 1]
```diff
@@ -66,6 +73,13 @@ const RecentDeliveriesObserverable = () => {
     }
   );
 
+
+
+
+
+
+
+
```
Comment (ao508) [inline src/pages/recentDeliveries/RecentDeliveriesPage.tsx:82]: clean up

File: src/pages/recentDeliveries/RecentDeliveriesPage.tsx [hunk 2]
```diff
@@ -184,10 +199,21 @@ const RecentDeliveriesObserverable = () => {
...
+            // jsdownload({CSVGenerate},"report.csv");
+          //  # jsdownload(data.requests,"blah.txt");
+          }}>Generate Report</Button>
+        </Col>
+      </Row> */}
+
+
+              <Col>
+          {/* <Button onClick={CSVGenerate}>Generate Report</Button> */}
+          <Button onClick={()=>{
```
Comment (ao508) [inline src/pages/recentDeliveries/RecentDeliveriesPage.tsx:214]: I think it's fine to still have the button callback function here even if it's still a work in progress. As long as it's not breaking the app while compiling

File: src/pages/recentDeliveries/csvExport.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+   export function CSVGenerate(requests) {
+        // let =requests
+        // let zarray=Object.entries(y)
+        // console.log(typeof(x))
+        // console.log(Object.entries(x))
+        // console.log(typeof(y))
+        // console.log(Object.entries(y))
+        // console.log(zarray)
+
```
Comment (ao508) [inline src/pages/recentDeliveries/csvExport.tsx:13]: clean up

File: src/pages/recentDeliveries/csvExport.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        ]
+
+        .map(e => e.join("\t")) 
+        .join("\n");
+
+        console.log(csvString)
+
+        jsdownload(csvString,"report.csv");
+        
+
```
Comment (ao508) [inline src/pages/recentDeliveries/csvExport.tsx:33]: it's a little hard to tell what's still in use here?

Could you do some clean up here?

**Review by ao508 — CHANGES_REQUESTED** [review]
Mostly suggestions for cleaning up bits of the code (see comments)

Additionally, please remove `package-lock.json` from this commit.

File: src/pages/recentDeliveries/RecentDeliveriesPage.tsx (hunk 2 above)
Comment (shalabhsuman) [reply src/pages/recentDeliveries/RecentDeliveriesPage.tsx:202]: keeping it as it is for now b/c Aaron and I are trying a couple of things here

File: src/pages/recentDeliveries/csvExport.tsx (hunk 4 above)
Comment (shalabhsuman) [reply src/pages/recentDeliveries/csvExport.tsx:26]: still a work in progress! trying to get the CSV-string right after we introduced the StaticTableColumn component.

**Pull request closed without merging**